site/docs/hosting/heroku.md

```diff
@@ -155,7 +155,7 @@ https://api.telegram.org/botabcd:1234/setWebhook?url=https%3A%2F%2Fgrammybot.her
 
 :::
 
-::: tip ⚡ Optimization (optional)
+::: tip ⚡ Optimization (Optional)
```

Comment:
For consistency with heading style, and as per https://github.com/grammyjs/website/pull/30#discussion_r735909521
```suggestion
::: tip ⚡ Optimization (optional)
```